recipes-openxt/sync-client/sync-client_git.bb

```diff
@@ -33,7 +33,19 @@ S = "${WORKDIR}/git"
 inherit distutils3
 inherit update-rc.d
 
-FILES_${PN} += "/etc/init.d/${INITSCRIPT_NAME}"
+PACKAGES += "${PN}-synccmd"
+
+FILES_${PN} = " \
+	${bindir}/sync-client \
+	${bindir}/sync-client-daemon \
+	${PYTHON_SITEPACKAGES_DIR}/sync_client/* \
+	/etc/init.d/${INITSCRIPT_NAME}"
+
+FILES_${PN}-synccmd = " \
+	${bindir}/pysynccmd \
+	${bindir}/sync-cmd \
+	${PYTHON_SITEPACKAGES_DIR}/pysynchronizer/* \
```

Comment:
I usually just list the directory `${PYTHON_SITEPACKAGES_DIR}/pysynchronizer`, but you can do it as you prefer.

Reply:
I can do either. If your suggestion is what is being done in the other recipes, it would be better to align for consistency's sake.